packages/auth-sveltekit/src/server.ts

```diff
@@ -132,28 +182,6 @@ export class ServerRequestAuth extends ClientAuth {
     return Auth.checkPasswordResetTokenValid(resetToken);
   }
 
-  private setVerifierCookie(verifier: string) {
-    const expires = new Date(Date.now() + 1000 * 60 * 24 * 7); // In 7 days
-    this.cookies.set(this.config.pkceVerifierCookieName, verifier, {
-      httpOnly: true,
-      sameSite: "lax",
-      path: "/",
-      expires,
-      secure: this.isSecure,
-    });
-  }
-
-  private setAuthCookie(authToken: string) {
-    const expires = Auth.getTokenExpiration(authToken);
-    this.cookies.set(this.config.authCookieName, authToken, {
-      httpOnly: true,
-      sameSite: "lax",
-      path: "/",
-      expires: expires ?? undefined,
-      secure: this.isSecure,
-    });
-  }
```

Comment:
Not a blocker, but the change here would be smaller and easier to follow for future endpoints if we _kept_ these class methods and just had them call the new function, so that other methods could continue to just call `this.setAuthCookie(tokenData.auth_token)`, etc.

```ts
  private setVerifierCookie(verifier: string) {
    setVerifierCookie(this.cookies, this.config, verifier);
  }

  private setAuthCookie(authToken: string) {
    setAuthCookie(this.cookies, this.config, authToken);
  }
```

And for bonus points doing a similar thing for the delete cookie functionality:

```ts
  private deleteVerifierCookie() {
    deleteCookie(this.cookies, this.config.pkceVerifierCookieName);
  }

  private deleteAuthCookie() {
    deleteCookie(this.cookies, this.config.authCookieName);
  }
```